### PR TITLE
Preserve governance outputs for losses

### PR DESCRIPTION
**Pull request by dansasser**
## Summary
- retain the final-layer policy logits and memory signals when aggregating governance outputs so downstream components can still reference them
- let the comprehensive biblical loss fallback to the new aggregated governance keys or full layer histories before computing the policy and memory penalties

## Testing
- python debug script to compute losses *(fails: torch unavailable in environment)*

------
https://chatgpt.com/codex/tasks/task_e_68cb50d84aa8832ea21cfdc2abddc079

### CODE REVIEW

File: SIM-ONE Training/prioritary_mvlm/advanced_losses.py [hunk 1]
```diff
@@ -461,22 +461,39 @@ def forward(
...
+            if memory_signals is not None:
                 # Memory efficiency - encourage sparse memory usage
-                memory_signals = governance_outputs['memory_signals']
-                if isinstance(memory_signals, list):
-                    memory_signals = memory_signals[-1]
                 memory_loss = torch.mean(torch.abs(memory_signals))
-            
+
             # Energy efficiency - L1 penalty on activations
             energy_loss = torch.mean(torch.abs(hidden_states))
```
[the remark this answers is not in the document]
Comment (dansasser) [reply SIM-ONE Training/prioritary_mvlm/advanced_losses.py:499]: @codex In SIM-ONE Training/prioritary_mvlm/advanced_losses.py around lines 497 to 499,
the energy_loss (L1 on hidden_states) is currently computed only when
governance_outputs is truthy; change this so energy_loss is computed
unconditionally by always assigning energy_loss =
torch.mean(torch.abs(hidden_states)). If governance_outputs was intended only to
scale or gate the contribution, keep any scaling/masking logic but do not skip
computing the loss; ensure hidden_states is defined in scope and that any
downstream aggregation uses energy_loss regardless of governance_outputs
presence.
